src/System.ServiceModel.Security/pkg/System.ServiceModel.Security.pkgproj

```diff
@@ -6,7 +6,7 @@
       <SupportedFramework>wp8</SupportedFramework>
     </ProjectReference>
     <ProjectReference Include="..\ref\System.ServiceModel.Security.csproj">
-      <SupportedFramework>net46;netcore50;dnxcore50</SupportedFramework>
+      <SupportedFramework>net45;netcore45;dnxcore50</SupportedFramework>
```

Comment:
@ericstj 
Same as above questions, I'm not sure by looking at the related .nuproj file and the S.SM.Http.pkgproj that we did together how we would know these changes are necessary. Can you clarify?